Extension/.vscode/tasks.json

```diff
@@ -95,9 +93,7 @@
             "command": "yarn",
             "args": [
                 "run",
-                "compile-watch",
-                "--loglevel",
-                "silent"
+                "compile-watch"
```

Comment:
You may want to take a look at this PR by Xinyu.  We had to fix an issue with the "watch" mode after upgrading webpack in CMake Tools.

https://github.com/microsoft/vscode-cmake-tools/pull/1685/files

Reply:
Ah, thanks. I see those errors now.

Reply:
I removed the flags that weren't working, but the watcher scripts don't work for me. Webpack gets silently stuck. Might be a webpack bug? I never use the watcher modes. Does anyone else care?

Reply:
I get `error: EPERM: operation not permitted, watch	at FSEvent.FSWatcher._handle.onchange ` ...seems like it's not our issue.

Reply:
I never use the watcher modes either.  Maybe just remove them?  If they're needed in the future, we can figure it out then.

Reply:
You have to change the background parser to make it work because webpack changed the messages.  That was the other part of Xinyu's PR.

Reply:
Ah, okay, that makes sense now.